akka-javasdk-maven/akka-javasdk-archetype/src/main/resources/archetype-resources/src/main/java/__packageInPathFormat__/application/package-info.java

```diff
@@ -0,0 +1,5 @@
+/**
+ * This module is for the application layer of the service. It is where the internal Akka components like entities and
+ * views should be defined.
+ */
+package ${package}.domain;
```

Comment:
Should be application package

Reply:
Dang. I'll fix

Reply:
https://github.com/akka/akka-sdk/pull/141